flexmeasures/data/services/sensors.py

```diff
@@ -1,43 +1,34 @@
 from __future__ import annotations
 
-from werkzeug.exceptions import NotFound
+import sqlalchemy as sa
 
 from flexmeasures import Sensor, Account
 from flexmeasures.data.models.generic_assets import GenericAsset
 
 
-def get_sensors(
-    account_name: str | None = None,
+def get_account_sensors(
```

Comment:
If we also get sensors that are not part of any account, the word "account" is misleading again :)